File: app/src/main/java/com/olq/multiple/LaunchActivity.java
package com.olq.multiple;

import android.content.Intent;
import android.os.Handler;

import com.olq.multiple.base.InitActivity;

public class LaunchActivity extends InitActivity {

    @Override
    public int getLayout() {
        return R.layout.activity_launch;
    }

    @Override
    public void onCreate() {
        Handler handler=new Handler();
        Runnable runnable=new Runnable() {
            @Override
            public void run() {
                startActivity(new Intent(getApplicationContext(),HomeActivity.class));
            }
        };
    }

}

File: app/src/main/kotlin/com.olq.multiple/LaunchActivity.kt
package com.olq.multiple

import android.content.Intent
import android.os.Handler
import com.olq.multiple.base.BaseActivity

/**
 * Created by Administrator on 2017/2/27.
 */
class LaunchActivity :BaseActivity(){


    override fun getLayout(): Int {
        return R.layout.activity_launch
    }

    override fun onCreate() {
        var handle=Handler();
        var run= Runnable { startActivity(Intent(application,HomeActivity::class.java)) }
//        handle.postDelayed(run,3000)
        handle.post(run)
    }

}